domains/transfer-agreements/TransferAgreements.API/API/Controllers/ConnectionInvitationsController.cs

```diff
@@ -33,6 +39,37 @@ public async Task<ActionResult> CreateConnectionInvitation()
 
         await connectionInvitationRepository.AddConnectionInvitation(newInvitation);
 
-        return Ok(new { connectionInvitationId = newInvitation.Id });
+        return CreatedAtAction(nameof(GetConnectionInvitation), new { id = newInvitation.Id }, new { connectionInvitationId = newInvitation });
```

Comment:
Why both new { id = newInvitation.Id } and new { connectionInvitationId = newInvitation }?